### ISSUE TITLE
when slurm file is provided, server values are not considered.

### ISSUE THREAD

**Issue by Avinash825** [cloudflare/gortr#53]
Hi, 

When we pass slurm file as an argument, Global repository values are not considered. Only the locally defined addresses in slurm file are considered. is this expected ?

thanks,
Avinash C

**Comment by lspgn** [comment]
Hello,
What do you mean by "global repository values"?
Can you provide a CLI example.

**Comment by Avinash825** [comment]
i mean, the values it fetches from web without any slurm file.... i have defined few ROA's in slurm file...
{
  "slurmVersion": 1,
  "validationOutputFilters": {
    "prefixFilters": [
    ],
    "bgpsecFilters": [
    ]
  },
  "locallyAddedAssertions": {
    "prefixAssertions": [

      {
        "asn": 13336,
        "prefix": "1.1.1.0/24",
        "maxPrefixLength": 30
      },

      {
        "asn": 13336,
        "prefix": "1.1.1.0/24",
        "maxPrefixLength": 30
      }
}
}

**Comment by lspgn** [comment]
Do you mean the values fetched from rpki.cloudflare.com/rpki.json?
Are the asserted ROAs not added?

**Comment by Avinash825** [comment]
Yes they are not added ... only locally defined values in slurm file are added..

**Comment by lspgn** [comment]
I ran the following
```
$ go run gortr.go -slurm ./myslurm.json
```
using the SLURM you posted (it had a JSON formatting issue):
```
{
  "slurmVersion": 1,
  "validationOutputFilters": {
    "prefixFilters": [],
    "bgpsecFilters": []
  },
  "locallyAddedAssertions": {
    "prefixAssertions": [
      {
        "asn": 13336,
        "prefix": "1.1.1.0/24",
        "maxPrefixLength": 30
      },
      {
        "asn": 13336,
        "prefix": "1.1.1.0/24",
        "maxPrefixLength": 30
      }
    ]
  }
}
```
Which returned me:

```
$ curl --silent 127.0.0.1:8080/rpki.json | jq '.roas[] | select(.prefix=="1.1.1.0/24")'
{
  "prefix": "1.1.1.0/24",
  "maxLength": 24,
  "asn": "AS13335",
  "ta": "Cloudflare - APNIC"
}
{
  "prefix": "1.1.1.0/24",
  "maxLength": 30,
  "asn": "AS13336"
}
{
  "prefix": "1.1.1.0/24",
  "maxLength": 30,
  "asn": "AS13336"
}
```

This should be the expected behavior? I'm not sure what you mean.
